app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -0,0 +1,196 @@
+
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import {
+  getConfig_nimbusConfig,
+  getConfig_nimbusConfig_channels,
+} from "../../types/getConfig";
+
+// TODO: find this doco URL
+const AUDIENCE_DOC_URL =
+  "https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FJT&title=Project+Nimbus";
+
+type FormAudienceConfig = Pick<
+  getConfig_nimbusConfig,
+  "channels" | "firefoxMinVersion" | "targetingConfigSlug"
+>;
+
+export const FormAudience = ({
+  experiment,
+  config,
+}: {
+  experiment: getExperiment_experimentBySlug;
+  config: FormAudienceConfig;
+}) => {
+  const {
+    channels,
+    firefoxMinVersion,
+    targetingConfigSlug,
+    populationPercent,
+    totalEnrolledClients,
+    proposedEnrollment,
+    proposedDuration,
+  } = experiment;
+
+  const channelsOptions = config.channels?.filter(
+    (item): item is getConfig_nimbusConfig_channels => item !== null,
+  );
+  const channelsDefaultValue = channels?.map((channel) =>
+    channelsOptions?.find((option) => option?.value === channel),
+  );
+
+  return (
+    <Form data-testid="FormAudience">
+      <Form.Group>
+        <Form.Row>
+          <Form.Group as={Col} controlId="channel" md={8} lg={8}>
+            <Form.Label>Channel</Form.Label>
+            <Select
+              isMulti
+              name="channel"
+              data-testid="channel"
+              defaultValue={channelsDefaultValue}
+              options={channelsOptions}
+            />
+          </Form.Group>
+          <Form.Group as={Col} controlId="minVersion">
+            <Form.Label>Min Version</Form.Label>
+            <Form.Control
+              as="select"
+              data-testid="minVersion"
+              defaultValue={firefoxMinVersion || ""}
+            >
+              <SelectOptions options={config.firefoxMinVersion} />
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} controlId="targeting">
+            <Form.Label>Advanced Targeting</Form.Label>
+            <Form.Control
+              as="select"
+              data-testid="targeting"
+              defaultValue={targetingConfigSlug || ""}
+            >
+              <SelectOptions options={config.targetingConfigSlug} />
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+      </Form.Group>
+
+      <Form.Group className="bg-light p-3">
+        <p className="text-secondary">
+          Please ask a data scientist to help you determine these values.{" "}
+          <LinkExternal href={AUDIENCE_DOC_URL}>Learn more</LinkExternal>
+        </p>
+
+        <Form.Row>
+          <Form.Group as={Col} className="mx-5" controlId="clientsPercent">
+            <Form.Label>Percent of clients</Form.Label>
+            <InputGroup className="mb-3">
+              <Form.Control
+                placeholder="0.00"
+                aria-describedby="clientsPercent-unit"
+                type="text"
```

Comment:
I think it could be nice to have the 4 inputs that only accept integers as `type="number"` for the up/down arrow selection and slightly better validation onblur. 

I'm guessing some of these should have a maximum allowed value too - we could create a quick follow up ticket for that and throw it into the maintenance fix version. Since max values and the links mentioned in another comment will come from collab with someone (probably Ana), might make sense to make one task for both?

Reply:
Planning on doing this when I add validation in the next issue (#3782)